main.tf

```diff
@@ -181,16 +184,24 @@ resource "google_cloud_run_v2_service" "creative_studio" {
   deletion_protection  = false
   iap_enabled          = !var.use_lb
   invoker_iam_disabled = !var.use_lb
-  launch_stage         = var.use_lb ? "GA" : "BETA"
+  launch_stage         = "BETA"
+  labels = {
```

Comment:
Labels are already added in the provider section. Hard coding the label values should be avoided since we don't know the correct values across all users.